### ISSUE TITLE
panic since v0.14.0

### ISSUE THREAD

**Issue by gnuletik** [wagslane/go-rabbitmq#169]
```
panic: runtime error: invalid memory address or nil pointer dereference
```

`handler.Mu` is never initialized:

https://github.com/wagslane/go-rabbitmq/blob/8e18cd3f8a038987656b793e529d5331d0b107ad/consume.go#L36

**Comment by astromechza** [comment]
Fix PR up here for now https://github.com/wagslane/go-rabbitmq/pull/170

**Comment by wagslane** [comment]
Fixed in v0.14.1
